src/modernizer/violations.xml

```diff
@@ -24,6 +24,12 @@
         <comment>Prefer String.toLowerCase(java.util.Locale)</comment>
     </violation>
 
+    <violation>
+        <name>java/io/File.toString:()Ljava/lang/String;</name>
+        <version>1.1</version>
```

Comment:
Why 1.1? It looks like this method has always been there: http://web.mit.edu/java_v1.0.2/www/javadoc/java.io.File.html#getPath()

Reply:
We use 1.1. for all. This is "day 0 for this file"